node/mocha.start.js

```diff
@@ -18,6 +18,7 @@ before(function () {
   }
   Test.sinon = require('sinon')
   Test.JSData = require('js-data')
+  Test.addAction = require('./dist/js-data-http-node').addAction
```

Comment:
It would be good to de-duplicate these 3 files. I would have preferred to do this via. es6 but I haven't gotten both build pipelines to work correctly with that =/